pandas/core/tools/datetimes.py

```diff
@@ -726,6 +579,11 @@ def parse_time_string(arg, freq=None, dayfirst=None, yearfirst=None):
     -------
     datetime, datetime/dateutil.parser._result, str
     """
+    res = tslib.parse_time_string(arg, freq, dayfirst, yearfirst)
```

Comment:
why is this needed?

Reply:
This wrapping belongs in tslib, will be fixed shortly.